CONTRIBUTING.md

```diff
@@ -17,15 +17,31 @@ Thanks for your interest in contributing code to the Oppia project! This page ex
 
 If you already know what you'd like to work on, you can skip this section.
 
-If you have an idea for something to do, first check if it's already been filed on the [issue tracker](https://github.com/oppia/oppia/issues). If so, add a comment to the issue saying you'd like to work on it, and we'll help you get started! Otherwise, please file a new issue and assign yourself to it.
+If you have an idea for something to do, first check if it's already been filed on the [issue tracker](https://github.com/oppia/oppia/issues). If so, add a comment to the issue saying you'd like to work on it, and we'll help you get started! Otherwise, please file a new issue.
 
-Otherwise, you could try searching the issue tracker for something that looks interesting! The issues have been categorized according to the type of help needed:
+Otherwise, you could try searching the issue tracker for something that looks interesting! High priority projects are tracked by [milestones](https://github.com/oppia/oppia/milestones), so we recommend you start there. The issues have been categorized according to the type of help needed:
```

Comment:
This was a problem in the previous version too, but I suggest changing the first sentence to "You could also try searching...", otherwise there are too many 'otherwise's :-)